Miktex: Fix redirect to FTP download URL

jlehtnie opened the pull request:
CTAN mirror URL may redirect to FTP server for some mirrors.
Fix redirect request handler to support FTP URI location.

aronovgj commented:
Thanks. `Invoke-WebRequest` is not Powershell 2 compatible. Any simple way to fix that?

jlehtnie committed:
Miktex: Fix redirect to FTP download URL

CTAN mirror URL may redirect to FTP server for some mirrors.
Fix redirect request handler to support FTP URI location.

jlehtnie commented:
Ah. That's why all the dotnet madness in the original script :)
The updated commit should be Powershell 2 compatible.

aronovgj commented:
Thanks
